app/models/hyrax/file_metadata.rb

```diff
@@ -2,15 +2,45 @@
 
 module Hyrax
   class FileMetadata < Valkyrie::Resource
+    GENERIC_MIME_TYPE = 'application/octet-stream'
+
+    ##
+    # Constants for PCDM Use URIs; use these constants in place of hard-coded
+    # URIs in the `::Valkyrie::Vocab::PCDMUse` vocabulary.
+    module Use
+      ORIGINAL_FILE = ::Valkyrie::Vocab::PCDMUse.OriginalFile
+      EXTRACTED_TEXT = ::Valkyrie::Vocab::PCDMUse.ExtractedText
+      THUMBNAIL = ::Valkyrie::Vocab::PCDMUse.Thumbnail
```

Comment:
In looking back through code trying to track down why case compare was used, I noticed that this should be...

```
THUMBNAIL_IMAGE = ::Valkyrie::Vocab::PCDMUse.ThumbnailImage
```

I can fix that if you want since I introduced it.

Reply:
ah. i'll add a patch here.